index.bs

```diff
@@ -5751,6 +5779,43 @@ and registration with IANA.</p>
 
 <p>Fragment identifiers have no meaning with <code>text/vtt</code> resources.</p>
 
+<h2 class="no-num" id=privacy-and-security-considerations>Privacy and Security Considerations</h2>
+
+
+<h3 id=fromat-security>Text-based format security</h3>
+
+<p>As with any text-based format, it is possible to construct malicious content that might cause
+buffer over-runs, value overflows (e.g. string representations of integers that overflow a given
+word length), and the like. Implementers should take care in implementing a parser that over-long
+lines, field values, or encoded values do not cause security problems.</p>
+
+<h3 id=styling-security>Styling-related security</h3>
+
+<p>WebVTT can embed style ‘snippets’, and they in turn can cause the loading of external style
+sheets through the use of a CSS “@import” rule, in user-agents that support CSS. Under these
+circumstances, the security and privacy considerations of CSS apply. In addition, it is possible for
+a user-agent to offer user style-sheets, and their presence and nature might be detectable by
+scripts running in the same user-agent (e.g. browser). This might enable fingerprinting of the user
+or reveal aspects of the user’s preferences (e.g. the choice of a large font size and/or
+high-contrast colors might indicate a user with visual impairments).</p>
+
+<h3 id=scripting-security>Scripting-related security</h3>
+
+<p>WebVTT does not include or enable scripting. However, it is possible to construct and deliver a
```

Comment:
Could also stress that it is important that user agents do not support a way to execute script embedded in a WebVTT file.